### ISSUE TITLE
Vessels shifting in orbit when Current Subspace Error reaches -5000ms and is reset to 0

### ISSUE THREAD

**Issue by JohannesMP** [godarklight/DarkMultiPlayer#110]
To illustrate the problem I'm sharing an example of what happened during a testing session yesterday:

---

I wanted to attempt building a station in orbit around minmus, in collaboration with a friend. The basic station design uses an 'H' shape, with two columns connected by a standard width spacer, with each column terminating in a senior docking port. All future components would attach using two docking ports, to allow for perfect alignment.

I've set up a private DMP Server on a VPS in the US. My ping (from Finland) is about 200ms, while my friend in Alaska gets a ping of about 80ms.

My friend launched the station core (named 'Vasnaxos'), and I wanted to bring on our first module; A small adapter that extends docking ports far from the station to allow for large ships:
<img src=http://i.imgur.com/BAB0Zqm.png >

It should be noted that at this point my friend was in the editor, so all observed latency was as a result of my own latency.

I had to go very slow since I had to attach it at the middle of this long piece. Here I am almost docked:
<img src=http://i.imgur.com/DAFayXJ.png >

However about every 45 seconds the station would jump by about a meter or so in its orbit. When I was just about to dock the two senior docking ports, the next jump resulted in the vessel intersecting the station:
<img src=http://i.imgur.com/gWvWCwu.png >

At this point I was locked in place, and could not do anything except wait for the next jump, which promptly shifted the station even more, and resulted in the intersecting parts breaking apart:
<img src=http://i.imgur.com/8JR9HwB.png >

Suffice it to say after spending a few hours already working on the modules this was pretty disheartening:
<img src=http://i.imgur.com/wUbQwoP.png >

---

So basically, this shouldn't happen ever. We only just started on the station, but this could obviously have happened on station we'd spent days building. I think when within 2.5km and other vessels, maintaining relative positions correctly should be the absolute top priority, over any sort of orbit accuracy.
